ngrinder-groovy/src/main/java/net/grinder/scriptengine/groovy/GroovyScriptEngineService.java

```diff
@@ -80,19 +76,7 @@ public GroovyScriptEngineService() {
 	 */
 	@Override
 	public List<Instrumenter> createInstrumenters() {
-
-		final List<Instrumenter> instrumenters = new ArrayList<>();
-
-		/*
-		 * if (!m_forceDCRInstrumentation) {
-		 * System.out.println("m_forceDCRInstrumentation is false."); // must using Instrumentation
-		 * }
-		 */
-		if (m_dcrContext != null) {
-			instrumenters.add(new JavaDCRInstrumenterEx(m_dcrContext));
-		}
-
-		return instrumenters;
+		return emptyList();

```

Comment:
We can use List.of() here

Reply:
nGrinder's project JDK is 1.8. So, cannot use `java.util.List.of()`.
We have two options for empty list
* `Collections.emptyList()`;
  it contains in `java.util.Collections`
* `ImmutableList.of()`;
  it contains in `org.python.google.common.collect.ImmutableList` <- it comes from `jython-standalone` dependency

Reply:
Ok.. just use emptyList()